storage-proofs-update/tests/common/mod.rs

```diff
@@ -1,118 +1,157 @@
+#![allow(dead_code)]
+
+use std::fs;
 use std::path::Path;
 
-use blstrs::Scalar as Fr;
-use filecoin_hashers::{Domain, Hasher};
+use ff::PrimeField;
+use filecoin_hashers::{Domain, Hasher, PoseidonArity};
 use generic_array::typenum::Unsigned;
-use merkletree::store::StoreConfig;
-
-use storage_proofs_core::{
-    merkle::{MerkleTreeTrait, MerkleTreeWrapper},
-    util::default_rows_to_discard,
+use merkletree::{merkle::get_merkle_tree_len, store::StoreConfig};
+use storage_proofs_core::merkle::{create_lc_tree, get_base_tree_count, split_config_and_replica};
+use storage_proofs_update::{
+    constants::{
+        TreeD, TreeDArity, TreeDDomain, TreeDHasher, TreeR, TreeRBase, TreeRDomain, TreeRHasher,
+    },
+    rho,
 };
-use storage_proofs_update::{constants::TreeRDomain, rho};
 
-// Selects a value for `h` via `h = hs[log2(h_select)]`; default to taking `h = hs[2]`.
-#[allow(dead_code)]
-pub const H_SELECT: u64 = 1 << 2;
+// Selects a value for `h` via `h = hs[log2(h_select)]`; default tests to use `h = hs[3]`.
```

Comment:
Note: because TreeR and TreeD are now hardcoded as `LCTree` and `DiskTree`, replace the `create_tree` helper with functions for creating TreeD and TreeR